src/screens/DiscoveryScreenV2/elements/Search.js

```diff
@@ -286,9 +302,8 @@ const styles = StyleSheet.create({
     marginTop: 0,
     zIndex: 10,
     height: dimen.size.DISCOVERY_HEADER_HEIGHT,
-    paddingTop: 7,
-    paddingBottom: 7,
-    borderBottomWidth: hideBackIcon ? 0 : 1,
+    paddingVertical: dimen.normalizeDimen(7),
+    borderBottomWidth: hideBackIcon ? 0 : dimen.normalizeDimen(1),
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

You have replaced hard-coded style values with dynamic ones using the `dimen.normalizeDimen` function. This is a good practice as it helps to maintain consistent UI across different screen sizes.
```diff
- paddingTop: 7,
- paddingBottom: 7,
- borderBottomWidth: hideBackIcon ? 0 : 1,
+ paddingVertical: dimen.normalizeDimen(7),
+ borderBottomWidth: hideBackIcon ? 0 : dimen.normalizeDimen(1),
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->